recipe/meta.yaml

```diff
@@ -15,14 +16,14 @@ package:
 source:
   fn: {{ fn }}
   url: https://pypi.io/packages/{{ pyver }}/{{ name[0] }}/{{ name }}/{{ fn }}
```

Comment:
@ocefpaf The pyver it needs is in the url here and it is not there because the bot doesn't understand the jinja2 at line 4.

Reply:
Can you just use `py` in the url?

Reply:
What is `py`? 3.8 or 38? It must be the latter and not the former. That is why we had the jinja string manipulation there.

Reply:
py is 38, but what the bot needs is an explicit listing of the urls for each selector. It cannot understand how to set pyver properly for each one.